frontend/src/scenes/funnels/FunnelBarChart/StepLegend.tsx

```diff
@@ -10,14 +10,33 @@ import { capitalizeFirstLetter, humanFriendlyDuration, percentage, pluralize } f
 import { ValueInspectorButton } from '../ValueInspectorButton'
 import { FunnelStepMore } from '../FunnelStepMore'
 import { userLogic } from 'scenes/userLogic'
+import { Noun } from '~/models/groupsModel'
 
-interface StepLegendProps extends ChartParams {
+type StepLegendProps = {
```

Comment:
only out of interest.... why convert from interface with extends to type with &

Reply:
Good catch! It doesn't make any difference here and could have stayed an interface. I was probably thinking I'd need to overwrite some existing property somehow.